docs/dev-guide.md

```diff
@@ -0,0 +1,39 @@
+# Dev Guide
+The guide provides steps to configure your local setup with running canary-checker
+
+
+## Setup a Kubernetes Cluster
+Before proceeding one must have access to a functioning Kubernetes cluster
+
+In this guide we'll be working with [kind](https://kind.sigs.k8s.io/)
+
+### Create a cluster
+
+- Install latest version of [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- Create the cluster with `kind create cluster --name <cluster-name>`
+
+### Install Metrics Server and Prometheus
+
+For installing and configuring metrics server and prometheus please see the [prereqs](prereqs.md)
+
```

Comment:
Please keep the spaces between paragraphs constant. At some stage I'll move us to a linting tool called markdownlint and we'll get automation on stuff like this.